delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -195,6 +197,79 @@ final class Files(
     } yield res
   }.span("createLink")
 
+  def copyFiles(
```

Comment:
Files being already quite long and complex, it feels like we could introduce a `BatchFiles` that would take care of all the validation and would pass along the commands to Files ?
It may also make the testing easier ?